python/ql/test/library-tests/frameworks/data/test.ql

```diff
@@ -75,6 +76,8 @@ class Sources extends ModelInput::SourceModelCsv {
         "testlib;;Member[CommonTokens].Member[Super].Subclass.Instance;test-source", //
         // method
         "testlib;;Member[CommonTokens].Member[Class].Instance.Method[foo];test-source", //
+        // testing non-positional arguments
+        "testlib;;Member[ArgPos].Member[MyClass].Subclass.Member[foo].Parameter[self];test-source", //
```

Comment:
> I'm interested to see whether the following MaD would target the parameter `arg`, or whether it would also go to `self`?

`Parameter[0]` targets `arg`, I've added a test.  

> next commits indicates that this was not the case when the code was originally written 

Yes, I seem to remember fixing those parameter indexes relatively late.